moesifwsgi/config_manager.py

```diff
@@ -21,6 +26,16 @@ def __init__(self, api_client, app_config, debug):
         self.current_etag = None
         self.config = None
         self.last_updated_time = datetime.utcnow()
+        self.__init_config__()
+
+    def __init_config__(self):
```

Comment:
We should not define double underscore methods in python.  The existing __ methods are special language keyword functions essentially.  This would just be def init_config(self):

Reply:
`__` is also used to classify a private function - helps developer to not use it directly outside class. Correct?

Reply:
Single underscore prefix is used for class private methods.  Double underscores on both sides are only for langauge methods like Class.__dict__ and __main__